client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -194,4 +204,11 @@ export class CommentNavigation extends Component {
 	}
 }
 
-export default localize( UrlSearch( CommentNavigation ) );
+const mapDispatchToProps = {
+	recordChangeFilter: status => composeAnalytics(
+		recordTracksEvent( COMMENTS_STATS_GROUP + '_change_filter', { status } ),
```

Comment:
Unfortunately `status` is not a prop, but a property of `navItems` (which is generated on the fly by the component itself).